core/include/drivers/clk.h

```diff
@@ -0,0 +1,193 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Bootlin
+ */
+
+#ifndef CLK_H
```

Comment:
The guard should be `__DRIVERS__CLK_H`

Reply:
Ok, I must say the include guards don't seems really homogeneous, I can see the following:
- #define __DRIVERS_GIC_H
- #define SCMI_MSG_H
- #define DRIVERS_WDT_H
Your proposal uses 2 _ after DRIVERS.
Which one should I use ?

Reply:
Sorry, my bad, I meant `__DRIVERS_CLK_H`. The important part here is a guard that is guaranteed unique without searching through the source tree first. Personally I prefer a `__` prefix in addition to that since it gets out of the way in other aspects too, so that's what I'd suggest.

Reply:
Ok, totally agree.